css/header.css

```diff
@@ -0,0 +1,117 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  max-width: 100vw;
```

Comment:
try using width.. check priority of max-width, width and min-width from documentation.